published-namespaces.txt

```diff
@@ -3,24 +3,8 @@ Android.Annotation
 Android.BillingClient
 Android.BillingClient.Api
 Android.Gms.Actions
-Android.Gms.Ads
-Android.Gms.Ads.AdManager
 Android.Gms.Ads.Afsn
 Android.Gms.Ads.Afsn.Search
-Android.Gms.Ads.AppOpen
-Android.Gms.Ads.Formats
-Android.Gms.Ads.H5
-Android.Gms.Ads.Initialization
-Android.Gms.Ads.Internal.Util.Client
-Android.Gms.Ads.Interstitial
-Android.Gms.Ads.Mediation
-Android.Gms.Ads.Mediation.CustomEvent
-Android.Gms.Ads.Mediation.Rtb
-Android.Gms.Ads.NativeAd
-Android.Gms.Ads.Preload
-Android.Gms.Ads.Query
-Android.Gms.Ads.Rewarded
-Android.Gms.Ads.RewardedInterstitial
```

Comment:
We noticed this last week, but we think something is wrong with some of the "Ads" libraries. We reverted the bump, because tons of APIs disappeared.

Did they introduce `-android` and `-jvm` versions of these packages and we need to add the `-android` one? Similar to SavedState.Android.

Reply:
> We noticed this last week, but we think something is wrong with some of the "Ads" libraries. We reverted the bump, because tons of APIs disappeared.

Gotcha.
 
> Did they introduce `-android` and `-jvm` versions of these packages and we need to add the `-android` one? Similar to SavedState.Android.

Possibly. Let me investigate.

Reply:
See: https://discord.com/channels/732297728826277939/732297837953679412/1413213645982138380

https://maven.google.com/web/index.html?q=com.google.android.gms#com.google.android.gms:play-services-ads-lite:24.3.0

> AdsLite aar is empty - no packages.
> Last time released on 2025-05-19
https://developers.google.com/android/guides/releases#may_19_2025

> The play-services-ads-lite is moving to a reduced release cadence, and is not distributed with this release. For more information, see the Lite SDK documentation.

> Lite SDK release cadence
https://developers.google.com/admob/android/lite-sdk#lite_sdk_release_cadence

> Seems like regular Ads and AdsLite are split now.
> Why is ads-lite empty I have no idea.